docs/marketplace/user/features/202106.0/merchant-products.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Merchant Products
```

Comment:
The feature is called "Marketplace Products", so please change the title and links.